pkg/binding/convert/postgresql_converter.go

```diff
@@ -27,5 +32,17 @@ func (m *PostgreSQLConnectionStringConverter) Convert(binding fileconfig.Service
 		addToParts("password", binding.Properties[keyPassword])
 	}
 	addToParts("dbname", binding.Properties[keyDatabase])
+
+	if binding.Properties[keyOptions] != "" {
+		// multiple options are expected to be separated by & (ampersand) sign, e.g. sslmode=disable&connect_timeout=10
+		options := strings.Split(binding.Properties[keyOptions], optionsSeparator)
+		for _, option := range options {
+			optionKeyValue := strings.Split(option, optionKeyValueSeparator)
+			if optionKeyValue != nil && len(optionKeyValue) == 2 {
+				addToParts(optionKeyValue[0], optionKeyValue[1])
```

Comment:
what if an option sting is "=xxx"?
a property with empty key doesn't sound right.

Reply:
please do a 'make lint' to do anything needs to be fixed.

Reply:
Done. Thanks for pointing this out - always good to run lint (should have checked the Makefile)